src/containers/Group/Suppliers/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component, PropTypes} from 'react';
+import Helmet from 'react-helmet';
+import { asyncConnect } from 'redux-async-connect';
+
+import { load as loadSuppliers } from 'redux/modules/suppliers/list';
+import List from './List';
+
+@asyncConnect([{
```

Comment:
@andresgutgon I guess I just need `@connect()` here instead of `@asyncConnect()`